pkg/sql/schemachanger/testdata/alter_table_add_column

```diff
@@ -914,12 +914,12 @@ notified job registry to adopt jobs: [2]
 test
 ALTER TABLE db.public.tbl ADD COLUMN k INT NOT NULL DEFAULT 42
```

Comment:
hmm. this is the 2nd test in this file, shouldn't this break because we only allow one test now?